--- 10-master.conf.orig	2019-03-14 20:28:23.079826002 +0000
+++ 10-master.conf	2019-03-14 20:31:13.504996014 +0000
@@ -5,6 +5,7 @@
 # intended to catch and kill processes that leak memory before they eat up
 # everything.
 #default_vsz_limit = 256M
+default_vsz_limit = 64M
 
 # Login user is internally used by login processes. This is the most untrusted
 # user in Dovecot system. It shouldn't have access to anything at all.
@@ -16,7 +17,7 @@
 
 service imap-login {
   inet_listener imap {
-    #port = 143
+    port = 0
   }
   inet_listener imaps {
     #port = 993
@@ -26,13 +27,13 @@
   # Number of connections to handle before starting a new process. Typically
   # the only useful values are 0 (unlimited) or 1. 1 is more secure, but 0
   # is faster. <doc/wiki/LoginProcess.txt>
-  #service_count = 1
+  service_count = 0
 
   # Number of processes to always keep waiting for more connections.
   #process_min_avail = 0
 
   # If you set service_count=0, you probably need to grow this.
-  #vsz_limit = $default_vsz_limit
+  vsz_limit = $default_vsz_limit
 }
 
 service pop3-login {
@@ -93,9 +94,11 @@
   }
 
   # Postfix smtp-auth
-  #unix_listener /var/spool/postfix/private/auth {
-  #  mode = 0666
-  #}
+  unix_listener /var/spool/postfix/private/auth {
+    group = postfix
+    mode = 0660
+    user = postfix
+  }
 
   # Auth process is run as this user.
   #user = $default_internal_user
